### ISSUE TITLE
Song 9950 Line 1 coded as 1

### ISSUE THREAD

**Issue by SamPassmore** [theglobaljukebox/cantometrics#28]
Hi Stella, 

@dorshilton has pointed out that song 9950 has line 1 coded as 1

@pesavage says songs in Cantometrics should never be coded as line_1 = 1 

Raw data shows this datapoint is coded as 34 which translates to 1 & 5
or No singers & A single predominant voice—a leader— stands out above the general effect of social unison.

Is it possible to check whether the original coding is correct? And if we need to correct the value? 

For reference the song is called "Olekwo'l" and is performed by the Yurok.

**Comment by SamPassmore** [comment]
My mistake - the song is titled "Curing song" or "Healing song". Olekwo'l is an ethnoymn.

**Comment by stellasilbert** [comment]
You're right, a coding of 1 & 5 does not make sense. I looked into this, and unfortunately this is a tricky one to solve. 

Coding IDs in the 9000s don't have scans of their original coding sheets in the LOC files, so I can't check the original coding. When I listened to the audio, it clearly did not match the song metadata or the coding (for example, according to metadata the song is supposed to have bells, and the audio has no instruments, yet it is coded as having instruments). I've done some investigating, and my conclusion is that the current audio is incorrect (it seems to be a different song from the same tape-- this is true of a few other songs on this tape too, which is also a problem that I need to fix). 

I should be able to get an audio file of the full tape from LOC and see if I can find the correct audio, and _then_ I can listen to it and figure out how to recode line 1. So this might take a few days but I'll try to get it sorted out asap. If for some reason I can't get the full audio, I would suggest either recoding line 1 to point 5 only, or removing this code, or leaving it with a note somewhere explaining the problem.

Have you checked the whole dataset to see if there are other cases like this, where line 1 is double coded as 1 and something else? I didn't think to check for that when I removed songs coded as 1. Let me know if you'd like me to check.

**Comment by SamPassmore** [comment]
Hi Stella,

If you can find the audio that would be great. I think you @annalwood and @pesavage are better placed than me to check what the code should be. 

Since we have the dataset coded as single codes now (in the CLDF format) these questions are easy to check.

I can confirm there are no other songs with line_1 = 1 in the dataset. 
I will add this as a test in the system. 

Do you know if there are any other lines that shouldn't have certain values? 
Currently there are tests to ensure that the values for each line exist as a possible code in the codebook for that line. But as in line 1 - there might be other situations where we want less codes than that. 

Sam
